gcc/config.gcc

```diff
@@ -4056,11 +4057,12 @@ case "${target}" in
 		# pick a default based on the ISA, preferring soft-float
 		# unless the D extension is present.
 		case "${with_abi}" in
-		ilp32 | ilp32f | ilp32d | lp64 | lp64f | lp64d)
+		ilp32 | ilp32f | ilp32d | lp64 | lp64f | lp64d | lp64q)
```

Comment:
ilp32e needs to be in this list of valid abis, otherwise you are overriding the user's choice, and won't give an error if the user makes a mistake with non-matching arch/abi.

Reply:
Fixed, that's my mistake when rebase with Q-extension...